jest.config.js

```diff
@@ -28,7 +28,6 @@ module.exports = {
   transform: {
     "^.+\\.(ts|tsx)$": "ts-jest",
   },
-  setupFiles: ["dotenv/config"],
```

Comment:
Doesn't seem like the unit tests need `dotenv` so removing which will get rid of these lovely errors we were seeing for every test 
>    console.error
      Error: ENOENT: no such file or directory, open '.env'